terraform/production/dynamodb.tf

```diff
@@ -17,3 +19,27 @@ resource "aws_dynamodb_table" "tenureinformationapi_dynamodb_table" {
     enabled = true
   }
 }
+
+data "aws_iam_policy_document" "assume_role" {
```

Comment:
Does the apply of this terraform config on the Prod DynamoDB happen straightaway or is there an manual approval gate it hits in the CI/CD pipeline?

Reply:
there is a manual approval and it only happens on the release branch